packages/clerk-js/src/core/auth/SessionCookieService.ts

```diff
@@ -52,33 +87,24 @@ export class SessionCookieService {
   }
 
   private async refreshSessionToken(): Promise<void> {
-    if (!inBrowser()) {
-      return;
-    }
-
```

Comment:
ℹ️ since this class will only be initialized in standard browsers, there is no need to guard for non browsers.